axum-extra/src/routing/mod.rs

```diff
@@ -25,6 +26,59 @@ pub use axum_macros::TypedPath;
 #[cfg(feature = "typed-routing")]
 pub use self::typed::{SecondElementIs, TypedPath};
 
+// Validates a path in compile time, used with the vpath macro.
+// Allow dead code needed because this is only used within the vpath macro
+// and if there's no call to it, then, there's no users of it.
```

Comment:
"Allow dead code"? I don't see `#[allow(dead_code)]` here, and I'm sure it's not necessary for public symbols anyways.